### ISSUE TITLE
Duplicate event

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#3707]
## User story 
As an event organizer, I want to be able to duplicate an event, so that I can host repeating events more easily.

> Just a little message to let you know that the ability to duplicate/clone an event would be extremely helpful. My organization
holds weekly events were folks are required to reserve a spot. The event is basically the same every time. So there you can see why this functionality would be super useful.

## MVP
A `duplicate` button on past events that copies everything except the date.

A setting to make an event repeat at a set time interval (similar to Google calendar)

**Comment by Betree** [comment]
Similar alternative, close to what was suggested for expenses in #3686: have a dropdown in the event creation form to select another event to use as a template.

**Comment by csg33k** [comment]
I would love this feature as well, We have regular yearly events that are very similar.  Things do change year to year but being able to create either a template or a clone feature would be great.  

Particularly when relating to tickets/tiers/donations etc.

**Comment by jdaviescoates** [comment]
Yes, this would be a very handy and time saving feature for me. I organise a fortnightly film and food event and most of the core details and ticket details are exactly the same every time. It would save me lots of time to not have to re-create the exact same tickets every single time.

**Comment by Betree** [comment]
We're currently [implementing](https://github.com/opencollective/opencollective-api/pull/9914) a feature for OCF collectives to duplicate their existing profile elsewhere. We'll be able to re-use this API to implement a "Duplicate event" functionality; we just need to figure out where in the interface to put it.
